Paws4Me/Paws4Me/Controllers/Logic/PetSingleDetailsViewController.swift

```diff
@@ -37,6 +36,9 @@ class PetSingleDetailsViewController: UIViewController {
             view.addSubview(petImageView)
         }
     }
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
Indentation looks a little bit off here 😄

Reply:
Thank you👍🏻